frontend/src/components/ShootDetails/ShootDetailsCard.vue

```diff
@@ -174,7 +174,11 @@ export default {
   mixins: [shootItem],
   computed: {
     expirationTimestamp () {
-      return this.shootAnnotations['shoot.garden.sapcloud.io/expirationTimestamp']
+      var expirationTimestamp = this.shootAnnotations['shoot.gardener.cloud/expiration-timestamp']
```

Comment:
return this.shootAnnotations['shoot.gardener.cloud/expiration-timestamp'] || this.shootAnnotations['shoot.garden.sapcloud.io/expirationTimestamp']